main.py

```diff
@@ -115,26 +115,30 @@ def process(data, factors):
             average[factor_set][surface] = np.average(np.array(average[factor_set][surface]), axis=0)
     return average
 
-def sort_directions(data):
+def choose_direction(data, average):
     user_surfaces = []
     for direction_name, surface_arrays in data.items():
-        user_surfaces.append((direction_name, len(surface_arrays.keys()),surface_arrays.keys()))
-    return sorted(user_surfaces, key=lambda x: x[1], reverse=True)
-
+        user_surfaces.append((direction_name, len(surface_arrays.keys())))
+    #get the number of surfaces stored in average as keys
+    user_surfaces = sorted(user_surfaces, key=lambda x: x[1], reverse=True)
+    chosen_surface_num = user_surfaces[0][1]
+    average_surface_num = len(average[next(iter(average))].keys())
+    #I am not sure if this is really necessary to check if they are equal
+    if average_surface_num == chosen_surface_num:
+        return user_surfaces[0][0]
+    
+           
 if __name__ == "__main__":
+    input("Factors in input_factors.ini should be expressed in %. For any factor without %, it will be returned as 1.\n Press any button to continue.")
```

Comment:
Is this necessary to display this every time some one running the code? We can, say, write README.md, or even better, to comment directly on input_factors.ini.

Reply:
Right right so I can only create a README.md in github? Can I also do it in git or pycharm?

Reply:
@harrywhoforks I'm just suggesting, you can keep the line intact if that's your intention

Reply:
As for the README.md I think it would be pointless if no one reads /knows how to read it, so just leave it alone.